assets/less/cds-rdm/globals/site.overrides

```diff
@@ -53,4 +53,46 @@
 
 .info-box {
   padding: @defaultPadding !important;
+}
+
+#records-list {
+
+  width: 100%;
+
+  h3, h4{
+    font-weight: normal;
+  }
+
+  .ui.grid {
+    margin: 0;
```

Comment:
Minor: you can replace this by using `m-0` instead and save a few lines here

Reply:
Same as below. In this case I'm a bit doubtful as maybe keeping the normal margin in one of the two grids could be a better idea.